app/resources/views.py

```diff
@@ -0,0 +1,61 @@
+from flask import render_template, redirect, url_for
+from flask.ext.login import login_required, current_user
+from . import resources
+from .. import db
+from ..models import Resource, ZIPCode, Address, User
+from .forms import ResourceForm
+
+
+@resources.route('/', methods=['GET'])
+def index():
+    return render_template('resources/index.html')
+
+
+@resources.route('/add', methods=['GET', 'POST'])
+@login_required
+def add():
+    form = ResourceForm()
+    if form.validate_on_submit():
+        # based on form's zip code, load or create ZIPCode and add to db
+        zip_code = ZIPCode.create_zip_code(form.postal_code.data)
+        # based on form's address, zip's id, load or create Address, add to db
+        street_num_route = str(form.street_number.data) + ' ' + form.route.data
+        # should create helper method in models/location.py
+        address = Address.query.filter_by(
```

Comment:
@aharelick should i add in this pr? also, this assumes that we should check if Address exists